src/test/java/org/broadinstitute/hellbender/tools/funcotator/FuncotatorIntegrationTest.java

```diff
@@ -1910,5 +1917,170 @@ public void testCanHandleSymbollicAlleleFuncotations() {
+                                                           final String dataSourcesPath,
+                                                           final String expectedOutputPath)
+    {
+
+        for ( final FuncotatorArgumentDefinitions.OutputFormatType outputFormatType :
+                FuncotatorArgumentDefinitions.OutputFormatType.values()) {
+            if (outputFormatType.equals(FuncotatorArgumentDefinitions.OutputFormatType.SEG)) {
+                continue;
+            }
+            final String typeCorrectedExpectedOutPath = FilenameUtils.removeExtension(expectedOutputPath) +
+                    "." + outputFormatType.toString().toLowerCase();
+            final File outputFile = createTempFile(tmpOutDir + File.separator + inputVcfName + ".funcotator", "." + outputFormatType.toString().toLowerCase());
+
+            final ArgumentsBuilder arguments = createBaselineArgumentsForFuncotator(
+                    inputVcfName,
+                    outputFile,
+                    referencePath,
+                    dataSourcesPath,
+                    referenceVersion,
+                    outputFormatType,
+                    true);
+            runCommandLine(arguments);
+        }
+
+    }
+
+    /** In this instance, we are again giving an already annotated file, but
+     * we are using the flag argument "true" to indicate that we want this file
+     * to be reannotated. There should be no errors.
+     */
+    @Test(dataProvider = "provideForCheckIfAlreadyAnnotatedTestWrong")
+    public void testCheckWhenWantReannotate(final String inputVcfName,
+                                                final String referencePath,
+                                                final String referenceVersion,
+                                                final String dataSourcesPath,
+                                                final String expectedOutputPath)
+    {
+
+        for ( final FuncotatorArgumentDefinitions.OutputFormatType outputFormatType :
+                FuncotatorArgumentDefinitions.OutputFormatType.values()) {
+            if (outputFormatType.equals(FuncotatorArgumentDefinitions.OutputFormatType.SEG)) {
+                continue;
+            }
+            final String typeCorrectedExpectedOutPath = FilenameUtils.removeExtension(expectedOutputPath) +
+                    "." + outputFormatType.toString().toLowerCase();
+            final File outputFile = createTempFile(tmpOutDir + File.separator + inputVcfName + ".funcotator", "." + outputFormatType.toString().toLowerCase());
+
+            final ArgumentsBuilder arguments = createBaselineArgumentsForFuncotator(
+                    inputVcfName,
+                    outputFile,
+                    referencePath,
+                    dataSourcesPath,
+                    referenceVersion,
+                    outputFormatType,
+                    true,
+                    Collections.emptyList(),
+                    true);
```

Comment:
Add comment noting that the `--reannotate-vcf` is being turned on here, to make that clearer for the reader of your code